File: 프로그래머스/unrated/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.js
function solution(numbers) {
    let arr = numbers.sort((a, b) => b - a);
    let num1 = arr[0] * arr[1];
    let num2 = arr[arr.length - 1] * arr[arr.length - 2];
    let num3 = arr[0] * arr[arr.length - 1];
    
    return Math.max(num1, num2, num3);
}

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
def solution(n):
    
    n.sort();
    num1 = n[0] * n[1];
    num2 = n[len(n) - 1] * n[len(n) - 2];
    num3 = n[0] * n[len(n) - 1];
    
    return max(num1, num2, num3)
